configs/components/pxp-agent.rb

```diff
@@ -152,7 +158,7 @@
   when "windows"
     # Note - this definition indicates that the file should be filtered out from the Wix
     # harvest. A corresponding service definition file is also required in resources/windows/wix
-    pkg.install_service "SourceDir\\#{settings[:base_dir]}\\#{settings[:company_id]}\\#{settings[:product_id]}\\service\\nssm.exe"
```

Comment:
#{settings[:product_id]} should remain, and we should really put nssm.exe in to puppet/bin.

The path to the bindir should now be:

SourceDir\\#{settings[:base_dir]}\\#{settings[:company_id]}\\#{settings[:product_id]}\\puppet\\bin

Reply:
Ok, thanks - it looks like the only other thing arriving in `servicedir` (`(prefix)/service`) is puppet's daemon.rb -- could that go in the bindir as well? Do you know if other things eventually use `(prefix)/service`, or if it could be gotten rid of?

Reply:
@caseywilliams I would install a 5.5.x package on both a windows and EL7 machine and compare things. I _think_ what you mentioned should be the only things in /service but I'm not positive.

We should also compare against EL7 to see where we put daemon.rb in linux packages